test/fixtures/semantic-html5-scenarios/revision-with-version-without-label.adoc

```diff
@@ -0,0 +1,4 @@
+= Document Title
+Author Name
+:revnumber: LPR55
+:version-label!:
```

Comment:
The preferred syntax for negating an attribute is to put the `!` before the name.

Reply:
🙃